### ISSUE TITLE
No cleanup method

### ISSUE THREAD

**Issue by northkode** [javve/list.js#396]
I'm interested in using this in a project.. but as of right now i see a memory leak waiting to happening. 
There is no var list new List().cleanup() which removes all event listeners that are created.

**Comment by javve** [comment]
Hmm, you have a point there. No one have mentioned that before but it seems reasonable to have a `.remove()` method if you add and remove a lot of List.js instances.

**Comment by northkode** [comment]
Yeah something like that would be awesome, cause I wanted to use this library in a mobile app but it would eventually leak memory with my list views. 
I took a look at what you were doing but realized I would probably miss something if I tried to tackle this myself without full knowledge of your code base.
